asab/api/service.py

```diff
@@ -24,6 +21,42 @@ def __init__(self, app, service_name="asab.ApiService"):
 		super().__init__(app, service_name)
 		self.WebContainer = None
 		self.ZkContainer = None
+		self.AttentionRequired = []  # list of errors found.
+
+	def attention_required(self, error_id=None):
```

Comment:
`def attention_required(self, att: dict, att_id=None)`

`att_id` - it is not necessarily an error.

`att` is an dictionary with "generic" information about attention request e.g.:

```
{
    "detail": "Network problem",
    "target_ip": "1.2.3.4".
}
```